src/Core/Addon/Theme/Theme.php

```diff
@@ -29,26 +29,42 @@
 use AbstractAssetManager;
 use Configuration;
 use PrestaShop\PrestaShop\Core\Addon\AddonInterface;
+use PrestaShop\PrestaShop\Core\Util\ArrayFinder;
 use PrestaShop\PrestaShop\Core\Util\File\YamlParser;
-use Shudrum\Component\ArrayFinder\ArrayFinder;
 
 class Theme implements AddonInterface
 {
+    /**
+     * @var ArrayFinder
+     */
     private $attributes;
 
-    public function __construct(array $attributes)
+    /**
+     * @param array $attributes Theme attributes
+     * @param string $configurationCacheDirectory Default _PS_CACHE_DIR_
+     * @param string $themesDirectory Default _PS_ALL_THEMES_DIR_
+     */
+    public function __construct(array $attributes, $configurationCacheDirectory = null, $themesDirectory = null)
```

Comment:
```suggestion
    public function __construct(array $attributes, string $configurationCacheDirectory = null, string $themesDirectory = null)
```

Reply:
This file Theme.php has been there for years and strict types are not enabled on it in order to preserve backward compatibility.

I am not sure we can use scalar types on such classes.